docs/modules/ROOT/partials/getting-started/ref-registry-rule-maturity-matrix.adoc

```diff
@@ -0,0 +1,44 @@
+// Metadata created by nebel
+// ParentAssemblies: assemblies/getting-started/as_registry-reference.adoc
+
+[id="registry-rule-maturity-matrix"]
+= {registry} content rule maturity matrix
+
+Not all rules are fully implemented for every Artifact Type supported by the registry.
+The following table documents the current maturity level for each rule and each
+artifact type.
+
+.{registry} content rule maturity matrix
+[%header,cols=3*] 
+|===
+|
+|VALIDITY
+|COMPATIBILITY
```

Comment:
Suggest adding header for "Artifact type", and removing all caps for "Validity" and "Compatibilty" headers